spec/models/logstash_queries/watcher_low_ctr_query_spec.rb

```diff
@@ -28,50 +28,50 @@
           "filter": [
             {
               "term": {
-                "params.affiliate": 'affiliate_name'
```

Comment:
The logstash query specs are an exception to our normal Rubocop rule that enforces single over double quotes. Double-quotes are preferable in these specs as they allow us to copy and paste directly into Kibana for development and troubleshooting.

Reply:
Didn't remember that ruby allows underscores in number literals -- can completely understand that that wouldn't play nicely with JSON.  Didn't know the latter dislikes single quotes, either.